src/gwt/panmirror/src/editor/src/nodes/rmd_chunk/rmd_chunk.ts

```diff
@@ -98,19 +97,31 @@ const extension = (
         },
 
         pandoc: {
-          codeBlockFilter: {
-            preprocessor: (markdown: string) => {
-              const md = markdown.replace(
-                /^([\t >]*```+)\s*(\{[a-zA-Z0-9_]+( *[ ,].*?)?\})([ \t]*\n)([\W\w]*?)(?:```)(?:[ \t]*)$/gm,
-                (_match: string, p1: string, p2: string, _p3: string, p4: string, p5: string, p6: string) => {
-                  return p1 + kRmdCodeChunkClass + '\n' + p2 + '\n' + p5 + '```\n';
-                },
-              );
-              return md;
-            },
-            class: kRmdCodeChunkClass,
-            nodeType: (schema: Schema) => schema.nodes.rmd_chunk,
-            getAttrs: () => ({ navigation_id: uuidv4() }),
+
+          blockCapsuleFilter: {
+            type: '3F175F2A-E8A0-4436-BE12-B33925B6D220',
+            match: /^([\t >]*)(```+\s*\{[a-zA-Z0-9_]+(?: *[ ,].*?)?\}[ \t]*\n[\W\w]*?\n[\t >]*```+)([ \t]*)$/gm,
```

Comment:
I think you said no to this yesterday, but, does knitr care if the numbers of opening and closing backticks match?

Reply:
Also, in the early part of the regex, why `>`?